### PR TITLE
dashboard-typescript

### PR DESCRIPTION
**Pull request by wolverineks**

### CODE REVIEW

File: legacy/config/config.d.ts [hunk 1]
```diff
@@ -62,3 +62,11 @@ type WalletConfig = {
 }
 
 export var getWalletConfigById: (walletImplementationId: string) => WalletImplementation
+
+// prettier-ignore
+export var getCardanoBaseConfig: (networkConfig: CardanoHaskellShelleyNetwork) => Array<{
+  StartAt?: number,
+  GenesisDate?: string,
+  SlotsPerEpoch?: number,
+  SlotDuration?: number,
```
Comment (blhossam) [inline legacy/config/config.d.ts:73]:
```suggestion
  startAt?: number,
  genesisDate?: string,
  slotsPerEpoch?: number,
  slotDuration?: number
```
properties on camelCase

File: src/Dashboard/DashboardNavigator.tsx [hunk 2]
```diff
@@ -1,29 +1,30 @@
...
+const Stack = createStackNavigator<{
+  'staking-dashboard': any
+  'staking-center': any
+  'biometrics-signing': any
+}>()
 
-const DelegationNavigatorSummary = () => {
-  const walletName = useSelector(walletNameSelector)
+export const DashboardNavigator = () => {
+  const walletName: string = useSelector(walletNameSelector)
```
Comment (wolverineks) [inline src/Dashboard/DashboardNavigator.tsx]: remove type cast

File: src/Dashboard/DashboardNavigator.tsx [hunk 3]
```diff
@@ -1,29 +1,30 @@
...
-import styles from '../TxHistory/styles/SettingsButton.style'
-import {Button} from '../UiKit'
-import StakingDashboard from './StakingDashboard'
-
-type DelegationNavigatorRoutes = {
-  'staking-dashboard': any,
-  'staking-center': any,
-  'biometrics-signing': any,
-}
+import iconGear from '../../legacy/assets/img/gear.png'
```
Comment (wolverineks) [inline src/Dashboard/DashboardNavigator.tsx]: import from src (typescript)

File: src/Dashboard/DashboardNavigator.tsx [hunk 4]
```diff
@@ -67,4 +68,8 @@ const DelegationNavigatorSummary = () => {
   )
 }
 
-export default DelegationNavigatorSummary
+const styles = StyleSheet.create({
```
Comment (wolverineks) [inline src/Dashboard/DashboardNavigator.tsx]: remove altogether

File: src/Dashboard/StakePoolInfos.tsx [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+
+  return {stakingStatus: query.data, ...query}
+}
+
+export const useStakePoolIds = (wallet: WalletInterface) => {
+  const {stakingStatus, ...stakingStatusQuery} = useStakingStatus(wallet)
+
+  return {
+    ...stakingStatusQuery,
+    stakePoolIds: stakingStatus?.poolKeyHash ? [stakingStatus.poolKeyHash] : undefined,
```
Comment (wolverineks) [inline src/Dashboard/StakePoolInfos.tsx]: return empty array

File: src/Dashboard/StakePoolInfo.tsx [hunk 6]
```diff
@@ -0,0 +1,151 @@
...
+          {strings.warning}
+        </Text>
+      </View>
+    </View>
+  ) : null
+}
+
+const useStakePoolInfo = (wallet: WalletInterface, stakePoolId: string) => {
+  const query = useQuery({
+    queryKey: [wallet.id, 'stakePoolInfo', stakePoolId],
```
Comment (wolverineks) [inline src/Dashboard/StakePoolInfo.tsx:57]: remove stakePoolId

File: src/Dashboard/Dashboard.tsx [hunk 7]
```diff
@@ -0,0 +1,261 @@
...
+            <UserSummary
+              totalAdaSum={utxoBalance}
+              totalRewards={accountBalance}
+              totalDelegated={totalDelegated}
+              onWithdraw={() => setShowWithdrawalDialog(true)}
+              disableWithdraw={wallet.isReadOnly}
+            />
+          </Row>
+
+          <VotingBanner onPress={() => navigation.navigate(CATALYST_ROUTES.ROOT)} disabled={false} />
```
Comment (wolverineks) [inline src/Dashboard/Dashboard.tsx]: remove disabled={false}

**Review by stackchain — APPROVED** [review]

**Review by Nebyt — APPROVED** [review]
I haven't noticed any new issues. :+1:

**Pull request merged**